charts/fluid/fluid/templates/controller/alluxioruntime_service.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.runtime.alluxio.statistics.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: fluid-pod-alluxioruntime-controller
```

Comment:
It's not suitable to be 'fluid-pod-alluxioruntime-controller', how about 'fluid-alluxioruntime-controller-ststic'

Reply:
Yes,I will fix.